docs/configuration.md

```diff
@@ -124,6 +122,31 @@ Now, create DNS records for the URLs configured above.
 
 ## Load Balancer on cloud-provider
 
+### AWS
+
+[Upload the required certificates](https://aws.amazon.com/premiumsupport/knowledge-center/import-ssl-certificate-to-iam/).  Create and configure `values/aws-ingress/demo-values.yaml` from the examples.
+
+```
+helm upgrade --install --namespace demo demo-aws-ingress wire/aws-ingress \
+    -f values/aws-ingress/demo-values.yaml \
+    --wait
+```
+
+To give your load balancers public DNS names, create and edit `values/external-dns/demo-values.yaml`, then run [external-dns](https://github.com/helm/charts/tree/master/stable/external-dns):
```

Comment:
There can only be a single external-dns chart installed (one per kubernetes cluster, not one per namespace). Add a line about this?